src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java

```diff
@@ -933,7 +934,8 @@ public class FlowStateAllowedEventService {
                 CHANGE_SOLICITOR_EMAIL,
                 LIP_CLAIM_SETTLED,
                 asyncStitchingComplete,
-                TRANSFER_ONLINE_CASE
+                TRANSFER_ONLINE_CASE,
+                INVALID_HWF_REFERENCE
```

Comment:
We need event in spec claim only. Not required here, please check

Reply:
done